build

```diff
@@ -8,7 +8,7 @@ clone_repo() { # <wd> <repo_name> <repo_uri> <repo_branch>
 }
 
 get_clone_uri() { # <wd> <repo_name>
-        grep -P "^$2;" "$1/repos.vendor" "$1/repos.custom" | sort -r | tail -1 | cut -d ":" -f 2,3 | cut -d ";" -f 2
+        cat "$1/repos.custom" "$1/repos.vendor" | grep -P "^$2;" | head -1 | cut -d ";" -f 2
```

Comment:
reverse files order and make tail (there could be the same repo twice on one file)
missing -h parameter for grep